src/Item.py

```diff
@@ -0,0 +1,42 @@
+# Assets for the items
+images = {
+    "sword": "assets/sword.png",
+    "spear": "assets/spear.png",
+    "shield": "assets/shield.png",
+    "gloves": "assets/gloves.png",
+}
```

Comment:
Now that we've make the game into a package, these will need to be updated, see [here](https://github.com/chanson02/DunwoodyDisaster/blob/45f61357e684e2417419c8be827babdf3e23ed76/dunwoody_disaster/views/fightScreen.py#L28).